test/unit/core.js

```diff
@@ -618,8 +618,12 @@ QUnit.test( "jQuery('html')", function( assert ) {
 
 	//equal( jQuery( "element[attribute=<div></div>]" ).length, 0,
 	//	"When html is within brackets, do not recognize as html." );
-	assert.equal( jQuery( "element:not(<div></div>)" ).length, 0,
-		"When html is within parens, do not recognize as html." );
+	if ( jQuery.find.compile ) {
+		assert.equal( jQuery( "element:not(<div></div>)" ).length, 0,
+			"When html is within parens, do not recognize as html." );
+	} else {
+		assert.ok( true, "Complex :not not supported in selector-native" );
```

Comment:
Is there a way to mark such assertions as skipped (e.g., `ok( true, … )` → `ok( "skip", … )`)? It won't bubble up to QUnit, but it might help us in the future.